lib/Interpreter/Interpreter.cpp

```diff
@@ -1034,6 +1037,21 @@ namespace cling {
     return name.startswith(utils::Synthesize::UniquePrefix);
   }
 
+  clang::SourceLocation Interpreter::getSourceLocation(bool skipWrapper) const {
+    const Transaction* T = getLatestTransaction();
+    if (!T)
+      return SourceLocation();
+
+    const SourceManager &SM = getCI()->getSourceManager();
+    if (skipWrapper) {
+      cling::ostrstream Strm;
+      Strm << "void " << utils::Synthesize::UniquePrefix << m_UniqueCounter
+           << "(void* vpClingValue) {\n ";
```

Comment:
The only way I saw to do this was refactor some stuff.
In particular removing the createUniqueWrapper (the optimization of inserting the name into the AST tree seems un-needed and pretty dependent on clang).
Removed some stuff that was duplicate code and not used in either cling or ROOT.